dbt_copilot_helper/commands/conduit.py

```diff
@@ -40,25 +39,27 @@ class CreateTaskTimeoutConduitError(ConduitError):
 CONDUIT_DOCKER_IMAGE_LOCATION = "public.ecr.aws/uktrade/tunnel"
 CONDUIT_ADDON_TYPES = [
     "opensearch",
-    "postgres",
+    "rds-postgres",
+    "aurora-postgres",
     "redis",
 ]
 
 
-def normalise_string(to_normalise: str) -> str:
-    output = re.sub("[^0-9a-zA-Z]+", "-", to_normalise)
-    return output.lower()
+def get_secret_name(addon_name: str) -> str:
```

Comment:
I'm not sure about the rename, it's still normalising a string, not actually getting a secret name from anyway. Maybe this if you want it to be specific to secret names...
```suggestion
def normalise_secret_name(addon_name: str) -> str:
```